tests/shakedown/shakedown/dcos/command.py

```diff
@@ -180,6 +202,18 @@ def run_command_on_agent(
     return run_command(host, command, username, key_path, noisy)
 
 
+@contextmanager
+def attached_cli():
+    """ Attaches the local dcos-cli to the clusters.
+
+    This assumes that DCOS_URL, DCOS_PASSWORD and DCOS_USERNAME are set.
+    """
+    # TODO: check if cluster already attached
```

Comment:
For the record, the new CLI is supposed to be idempotent in the setup operation.

Reply:
Nice. So I can remove the todo. Thanks!